cal/api/views.py

```diff
@@ -201,14 +169,17 @@ class ColorCategoryDetailEvents(generics.ListAPIView):
     serializer_class = GEventSerializer
 
     def get_queryset(self):
-        return ColorCategory.objects.get(user=self.request.user, id=self.kwargs['pk']).query()
+        category = ColorCategory.objects.get(user=self.request.user, id=self.kwargs['pk'])
+        return category.query()
```

Comment:
I don't think there is a good reason to change this from what it was before. The one liner is cleaner than setting a temporary variable